### ISSUE TITLE
std parsing for -verificationLogger:text

### ISSUE THREAD

**Issue by seebees** [dafny-lang/dafny-reportgenerator#20]
I want to be able to run with `-verificationLogger:text`
So that I can investigate what is slow or expensive.
The CSV report does not have as much information.

But `text` output has _everything_ and it is hard to filter.
It would be great to be able to use the same tool.

Another option would be to add the information to the CSV report.
There could then be options to ONLY print out exced the limit!

**Comment by robin-aws** [comment]
Yeah, I think the right direction is adding the same information to the CSV report, which might require restructuring the output into multiple linked files (since the data isn't just a flat table any more).
